sh/tests/solutions/in-back-ground.sh

```diff
@@ -0,0 +1 @@
+nohup cat facts | grep "moon" && echo "moon was found" >sol-output.log 2>&1 &
```

Comment:
the `echo` message is not consistent with the subject: either change the required success message in the log stated in the subject, or change it here as it is now in the subject:  `The moon fact was found!`

Reply:
done